python/servo/platform/macos.py

```diff
@@ -54,8 +54,10 @@ def is_gstreamer_installed(self, cross_compilation_target: Optional[str]) -> boo
             return False
         return True
 
-    def _platform_bootstrap(self, _force: bool) -> bool:
-        return self._platform_bootstrap_gstreamer(False)
+    def _platform_bootstrap(self, force: bool) -> bool:
+        installed_something = self._platform_bootstrap_gstreamer(False)
+        installed_something |= self.taplo_install(force)
+        return installed_something
```

Comment:
Instead of explicitly calling `taplo_install(...)` here, let's call `taplo_install(...)` in the version of `bootstrap()` in `base.py`.